inputs/torus.pin

```diff
@@ -100,7 +103,7 @@ recon = weno5
 c2p_max_iter = 100
 c2p_tol = 1.e-8
 c2p_method = robust
-Ye = false
+Ye = true
```

Comment:
maybe still set default to false so as to not mess up the ideal gas run. I guess Ye doesn't do anything in that case.